sdk/storage/storage-file/test/node/highlevel.node.spec.ts

```diff
@@ -290,14 +255,14 @@ describe("Highlevel", () => {
     assert.ok(eventTriggered);
   });
 
-  it("bloburl.download should success when internal stream unexcepted ends at the stream end", async () => {
-    await uploadFileToAzureFile(Aborter.none, tempFileSmall, fileClient, {
+  it("blobClient.download should success when internal stream unexcepted ends at the stream end", async () => {
```

Comment:
Thanks for catching this.  We should use `fileClient` instead though.

Reply:
Oh yeah, thanks.